application/build.gradle

```diff
@@ -55,10 +47,9 @@ dependencies {
   implementation 'org.flywaydb:flyway-core'
   implementation 'org.flywaydb:flyway-database-postgresql'
   implementation 'software.amazon.awssdk:cognitoidentityprovider'
-  implementation "de.siegmar:logback-awslogs-json-encoder:${logbackAwslogsJsonEncoderVersion}"
-  implementation "io.github.wimdeblauwe:error-handling-spring-boot-starter:${errorHandlingSpringBootStarterVersion}"
+  implementation "de.siegmar:logback-awslogs-json-encoder:2.0.0"
+  implementation "io.github.wimdeblauwe:error-handling-spring-boot-starter:4.5.0"
 
-  developmentOnly 'org.springframework.boot:spring-boot-devtools'
```

Comment:
there's an issue with the Dynamo AWS client and the Devtools, see https://github.com/aws/aws-sdk-java-v2/issues/2604

I couldn't get it working and the dynamo db feature still seems to be broken, either it can't find the table or there's a class cast exception. Feel free to investigate it further.